### DISCUSSION TITLE
experiment annotation

### DISCUSSION THREAD

**Discussion by wikeray** [Nesvilab/FragPipe-Analyst-doc#68] [Q&A]
I have tried several different ways to format the experiment annotation for FragPipe Analyst but nothing is working.  I have used this software in the past without a problem.  What is the correct way to format the experiment annotation for the most recent version of FragPipe Analyst?

**Comment by hsiaoyi0504** [comment]
@wikeray We didn't change the way of experiment annotation recently. https://fragpipe-analyst-doc.nesvilab.org/Formatting.html does this help?
